### DISCUSSION TITLE
get the source -> target mapping in the lineage-holder module

### DISCUSSION THREAD

**Discussion by JustinJanz** [reata/sqllineage#315] [Q&A]
Hi Thanks for this wonderful library. I am trying to use this library to generate a lineage graph from some dmls but i am unable to get the column level lineage as a set of nodes including the intermediate tables. I am currently getting the column lineage using the function  get_column_lineage() and getting the source and target columns with lineage. Can you please tell me if there is a way to get the details that is held in the lineage_holder  module as a digraph in the form of a json output or in some fashion as that.

**Comment by reata** [comment]
Sorry I missed this thread. As of v1.3.7, there's no dedicated to_json functionality. As a walk-around, you can use the to_cytoscape method (https://github.com/reata/sqllineage/blob/v1.3.7/sqllineage/runner.py#L88) in LineageRunner and passing level=LineageLevel.COLUMN. This data structure is for frontend library Cytoscape.js and is also in json format.
